doc/src/fix_bond_create.rst

```diff
@@ -110,7 +116,16 @@ actually created.  The *fraction* setting must be a value between 0.0
 and 1.0.  A uniform random number between 0.0 and 1.0 is generated and
 the eligible bond is only created if the random number < fraction.
 
-Any bond that is created is assigned a bond type of *bondtype*
+The *aconstrain* keyword allows to specify a minimal and maximal angle
```

Comment:
I think this paragraph is the place to say that the aconstrain keyword only works with the fix bond/create/angle command, as in the example above.